### DISCUSSION TITLE
post/create-an-environment-where-you-can-grow/

### DISCUSSION THREAD

**Discussion by giscus** [jgjgill/blog#5] [Comments]
# post/create-an-environment-where-you-can-grow/

2023년 6월 회고 - 성장할 수 있는 환경 만들기

https://jgjgill.com/post/create-an-environment-where-you-can-grow/

**Comment by colorkite10** [comment]
원하시는 바를 향해 잘 달려가고 계신 것 같네요! 멋지십니다!

**Comment by 1eecan** [comment]
회고글을 보고 어떤 고민을 했고, 해결을 어떻게 했으며 개선할 방향까지 알 수 있었습니다!
글 솜씨가 뛰어나시네요🙂🙂 7월달도 화이팅!!

**Comment by MinwooP** [comment]
읽으면서 느끼는 점이 많은 회고글이네요 !! 자극 많이 받고 갑니다 ㅎㅎㅎ 7월도 화이팅입니다 종길님 !
